source/sessions/tests/unified/snapshot-sessions-not-supported-server-error.json

```diff
@@ -6,7 +6,8 @@
       "minServerVersion": "3.6",
       "maxServerVersion": "4.4.99",
       "topologies": [
-        "replicaset, sharded-replicaset"
+        "replicaset",
+        "sharded-replicaset"
```

Comment:
Should we include "single" here as well?

Reply:
Yes we should test against all topologies. 3.6 standalones report this expected error:
```
{'ok': 0.0, 'errmsg': "readConcern.level must be either 'local', 'majority', 'linearizable', or 'available'", 'code': 9, 'codeName': 'FailedToParse'}
```
4.0+ standalones report:
```
 {'ok': 0.0, 'errmsg': 'read concern level snapshot is only valid in a transaction', 'code': 72, 'codeName': 'InvalidOptions'}
 ```

Reply:
Should we also run this test against the combination of 5.0+ and "single"?

Reply:
Good call. 5.0+ standalones report:
```
{'ok': 0.0, 'errmsg': 'node needs to be a replica set member to use readConcern: snapshot', 'code': 123, 'codeName': 'NotAReplicaSet'}
```